package.json

```diff
@@ -7,7 +7,7 @@
     "================================================================================"
   ],
   "name": "@aurodesignsystem/auro-accordion",
-  "version": "3.2.0-beta.1",
+  "version": "3.2.1",
```

Comment:
This should till be the beta version because that "is" what is in the beta branch right now. The semantic release will set this to the right value when we merge this from beta into main.